xxhash.h

```diff
@@ -1403,20 +1403,7 @@ XXH3_128bits_reset_withSecretandSeed(XXH3_state_t* statePtr,
 
 #ifndef XXH_FORCE_MEMORY_ACCESS   /* can be defined externally, on command line for example */
    /* prefer __packed__ structures (method 1) for gcc on armv7+ and mips */
-#  if !defined(__clang__) && \
-( \
-    (defined(__INTEL_COMPILER) && !defined(_WIN32)) || \
-    ( \
-        defined(__GNUC__) && ( \
-            (defined(__ARM_ARCH) && __ARM_ARCH >= 7) || \
-            ( \
-                defined(__mips__) && \
-                (__mips <= 5 || __mips_isa_rev < 6) && \
-                (!defined(__mips16) || defined(__mips_mips16e2)) \
-            ) \
-        ) \
-    ) \
-)
+#  ifdef __GNUC__
```

Comment:
This will impact both `gcc` and `clang`. 
Previously, this mode was only activated for `gcc`.
Is it an intentional objective ? 
Is this mode desirable only for `gcc`, or for both `gcc` and `clang` even though `clang` seems to work fine with `memcpy()` (method 0)?